docs/releasehistory.md

```diff
@@ -8,6 +8,19 @@ Releases follow the `major.minor.micro` scheme recommended by [PEP440](https://w
 
 ## Current Development
 
+### Behaviors changed
+
+- [PR #1021](https://github.com/openforcefield/openforcefield/pull/1021): Renames
+  [`openff.toolkit.utils.exceptions.ParseError`](openff.toolkit.utils.exceptions.ParseError) to
```

Comment:
In general I'm pretty sloppy with release notes between releases, since Jeff and/or Josh will clean them up before making an actual release.

But in this case it doesn't really make sense to try to link to an exception that no longer exists, does it?